src/content/operations/deployment/subctl/_index.en.md

```diff
@@ -250,6 +250,32 @@ When running `benchmark latency`, two types of tests will be executed:
 | `--verbose`                         | Produce verbose logs during benchmark tests
 <!-- markdownlint-enable line-length -->
 
+### `validate`
+
+The `subctl validate` is a diagnostic tool that can be used to validate the Submariner deployment.
+
+This tool can be helpful to detect a specific problem when `subctl verify` failed.
```

Comment:
```suggestion
This tool can be helpful to detect specific problems when some aspect of the Submariner deployment isn't working properly.
```

Reply:
Thanks for the review comments @tpantelis. 

Broadly, there are two types of validation this tool does 
1. If there is some issue with Submariner deployment itself.
2. If there are any configuration issues in the Cluster because of which Submariner functionality will be affected.

Can you please suggest accordingly.

Reply:
I updated my comment.  @dfarrell07 Wdyt?

Reply:
LGTM :+1: